src/Npgsql/NpgsqlCommand.cs

```diff
@@ -960,7 +960,7 @@ internal void ProcessRawQuery(SqlQueryParser? parser, bool standardConformingStr
             break;
 
         default:
-            ThrowHelper.ThrowInvalidOperationException($"Internal Npgsql bug: unexpected value {CommandType} of enum {nameof(CommandType)}. Please file a bug.");
+            ThrowHelper.ThrowInvalidOperationException($"Internal Npgsql bug: unexpected value {{0}} of enum {nameof(CommandType)}. Please file a bug.", CommandType);
```

Comment:
We can just throw ArgumentOutOfRangeException for unhandled enum values (it's basically an assertion).